.github/workflows/update-viablestrict.yml

```diff
@@ -0,0 +1,25 @@
+name: Update viable/strict
+
+on:
+  schedule:
+    - cron: 17,47 * * * *
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  do_update_viablestrict:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    runs-on: ubuntu-20.04
+    environment: ${{ (github.event_name == 'schedule') && 'update-viable-strict' || '' }}
+    steps:
+      - name: Update viable/strict
+        uses: pytorch/test-infra/.github/actions/update-viablestrict@main
+        with:
+          repository: pytorch/executorch
+          stable-branch: viable/strict
+          requires: '[\"pull\", \"lint\"]'
```

Comment:
Should we include doc build as well?

Reply:
We could, we can add that if a successful doc build is considered important enough for the stable branch.  Let me add that, if it turns out not needed, it can be removed.